python/tank/authentication/ui/qt_abstraction.py

```diff
@@ -12,9 +12,45 @@
 Imports Qt without having to worry whether we are using PyQt4 or PySide.
 """
 try:
-    from PySide import QtCore, QtGui
-except ImportError:
+    from PySide2 import QtCore, QtWidgets, QtGui
+    # FIXME:  We need to refactor the PySide2 patching code from sgtk.platform.qt. This will get
+    # us going in the meantime.
+
+    class _ModuleHack(object):
```

Comment:
YIKES!!! Don't worry, this is there to unblock Jeff ASAP so he doesn't need to tweak core.